### DISCUSSION TITLE
Only update files that have no metadata.

### DISCUSSION THREAD

**Discussion by Deejay85** [Difegue/LANraragi#533] [Q&A]
I got a lot of files whose metadata I would like to work on, but the list is so long, it's going to be a little more than unreasonable to have it all taken care of in one go (91044 and still growing). As a result is there a way to have only the files that have no metadata to be processed? I'm not sure how hard it would be since I imagine a log of what has or has not had any metadata added to it shouldn't be that hard to do.

**Comment by Difegue** [comment]
If you're using batch tagging, the archives that don't have any metadata should be pre-checked in the list when loaded.

Apart from that, you can get the raw list of untagged archives through /api/archives/untagged, IIRC.

Let me know if you need something else that isn't provided by those existing features.

**Comment by Deejay85** [comment]
Can you please give me the name of the file I'm looking for? I tried finding the file myself by clicking on "open app folder," but couldn't find anything that would be relevant to what I'm looking for.

**Comment by Difegue** [comment]
Hmm, you don't need to look for a specific file for this: The Batch Tagging feature is available through a link at the top of the app's webpage:  

![](https://files.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LPbsOo9_KVq_qSlpg5H%2Fsync%2F30483b177e323cd38a08ee58458d7ebf1fb77bef.jpg?generation=1611241081359982&alt=media)

As for the API call, you can use it through your browser, with an URL like this:
`http://localhost:3000/api/archives/untagged`
